:

```diff
@@ -0,0 +1 @@
+i
```

Comment:
According current code layout , it doesn't look that any  other flow parser except "Type_PacketIn" is doing building match filed TLV processing and  issue is cause because of match rule processing in context of "Type_PacketIn" parser event. As of now, we can consider this issue is isolated to "Type_PacketIn" flows. 
===
        case Type_PacketIn:
                message = new(PacketIn)
                err = message.UnmarshalBinary(b)
==
func (p *PacketIn) UnmarshalBinary(data []byte) error {
        err := p.Header.UnmarshalBinary(data)
        n := p.Header.Len()

        p.BufferId = binary.BigEndian.Uint32(data[n:])
        n += 4
        p.TotalLen = binary.BigEndian.Uint16(data[n:])
        n += 2
        p.Reason = data[n]
        n += 1
        p.TableId = data[n]
        n += 1
        p.Cookie = binary.BigEndian.Uint64(data[n:])
        n += 8

        if err := p.Match.UnmarshalBinary(data[n:]); err != nil {
                return err
        }
        n += p.Match.Len() . ====> before fix, core is happening  at this place on referencing nil .